x-pack/plugins/enterprise_search/public/applications/workplace_search/views/groups/components/group_source_prioritization.tsx

```diff
@@ -54,7 +54,7 @@ const ZERO_STATE_TITLE = i18n.translate(
 const ZERO_STATE_BUTTON_TEXT = i18n.translate(
   'xpack.enterpriseSearch.workplaceSearch.groups.sourceProioritization.zeroStateButtonText',
   {
-    defaultMessage: 'Add shared content sources',
+    defaultMessage: 'Add organizational content sources',
```

Comment:
Here is an example of a button with correct casing, RE: the earlier comment about casing

Reply:
Thanks!